application/wlp-web-java-spring/src/test/java/webapp/tier/util/BeforeAllTest.java

```diff
@@ -0,0 +1,63 @@
+package webapp.tier.util;
+
+import static org.junit.Assert.fail;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+import javax.jms.ConnectionFactory;
+import javax.jms.MessageConsumer;
+import javax.jms.MessageProducer;
+import javax.jms.Queue;
+import javax.jms.Session;
+import javax.jms.Topic;
+import javax.sql.DataSource;
+
+import org.mockito.ArgumentMatchers;
+import org.springframework.mock.jndi.SimpleNamingContextBuilder;
+
+@SuppressWarnings("deprecation")
+public class BeforeAllTest {
```

Comment:
Codacy found an issue: [The utility class name 'BeforeAllTest' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6170659)